server/auth/authApiMiddleware.js

```diff
@@ -0,0 +1,34 @@
+const { checkCountryAccessFromReqParams, checkAdminAccess } = require('../utils/accessControl')
+const { getUser } = require('../utils/requestUtils')
+const { allowedToEditDataCheck } = require('../assessment/assessmentEditAccessControl')
+
+const requireCountryEditPermission = async (req, res, next) => {
+  const section = req.params.section || req.query.section
+  const countryIso = req.params.countryIso || req.query.countryIso
```

Comment:
in RequestUtils we could add a method like:
```
export const getParams = req =>
  R.pipe(
    R.mergeLeft(R.prop('query', req)),
    R.mergeLeft(R.prop('params', req)),
    R.mergeLeft(R.prop('body', req)),
    // Convert String boolean values to Boolean type
    R.mapObjIndexed(val => R.ifElse(v => v === 'true' || v === 'false', R.always(val === 'true'), R.identity)(val)),
  )({})
```
so that your code above would become:
```
const {countryIso, section} = Request.getParams(req)
```

what do you think?